chain/challenges-10.test.js

```diff
@@ -0,0 +1,198 @@
+
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named count that, given an integer and an array of arrays, uses either
+// filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+//
+// Note: You might need to use the same method more than once.
+//
+// For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+// ------------------------------------------------------------------------------------------------
+
+const count = (target, input) => {
+  // Solution code here...
+  let answer = 0;
+  input.map( ele => {
```

Comment:
You're using `map` here, but since you're not trying to generate another array of arrays, it would be more idiomatic to use `forEach`.